packages/build-tools/src/android/expoUpdates.ts

```diff
@@ -44,7 +44,11 @@ export async function androidSetChannelNativelyAsync(ctx: BuildContext<Job>): Pr
 export async function androidSetClassicReleaseChannelNativelyAsync(
   ctx: BuildContext<Job>
 ): Promise<void> {
-  assert(ctx.job.releaseChannel, 'releaseChannel must be defined');
+  const { releaseChannel } = ctx.job;
+  if (!releaseChannel) {
+    throw Error('releaseChannel must be defined');
+  }
```

Comment:
```suggestion
  const { releaseChannel } = ctx.job
  assert(releaseChannel, 'releaseChannel must be defined');
```

It's just a semantic difference, but assert should've been used to guard against programmer error and exceptions from user errors/external issues. Here if releaseChannel is null then it's fault of the code that is calling this function